ga4-data/live/language.json

```diff
@@ -50,7 +50,6 @@
   "totals": {
     "visits": 3550587183,
     "languages": {
-      "(other)": 41260289,
```

Comment:
I'd consider keeping this in the data here and dealing with removal in a different way. If you delete this here it is only going to impact local - it will still be in the bucket on develop env.

Reply:
Good point, will add a filter in the js to remove this key. I see that done in other block charts.